stdlib/collections/__init__.pyi

```diff
@@ -286,9 +286,9 @@ class defaultdict(Dict[_KT, _VT], Generic[_KT, _VT]):
 class ChainMap(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     maps: list[Mapping[_KT, _VT]]
     def __init__(self, *maps: Mapping[_KT, _VT]) -> None: ...
-    def new_child(self, m: Mapping[_KT, _VT] | None = ...) -> ChainMap[_KT, _VT]: ...
+    def new_child(self: _S, m: Mapping[_KT, _VT] | None = ...) -> _S: ...
     @property
-    def parents(self) -> ChainMap[_KT, _VT]: ...
+    def parents(self: _S) -> _S: ...
```

Comment:
Nit: There's a `_typeshed.Self` type alias, which we prefer for those self types (as a marker).

Reply:
Fixed, I originally based it off defaultdict's `copy()` which I guess was missed when `_typeshed.Self` was introduced.